src/components-examples/material/form-field/form-field-custom-control/example-tel-input-example.html

```diff
@@ -1,7 +1,10 @@
 <div [formGroup]="parts" class="example-tel-input-container">
-  <input class="example-tel-input-element" formControlName="area" size="3" aria-label="Area code" (input)="_handleInput()">
+  <input class="example-tel-input-element" formControlName="area"
```

Comment:
I think in other places we put each attribute on its own line. E.g.

```
<input 
  class="example-tel-input-element" 
  formControlName="area"
  size="3" 
  aria-label="Area code" 
  (input)="_handleInput()">
```

Reply:
Okay, I will follow the same pattern and put each attribute to its own line.